packages/cli/src/cmds/beacon/initBeaconState.ts

```diff
@@ -17,6 +17,26 @@ import {downloadOrLoadFile} from "../../util";
 import {IBeaconArgs} from "./options";
 import {defaultNetwork, IGlobalArgs} from "../../options/globalOptions";
 import {getGenesisFileUrl} from "../../networks";
+import {weakSubjectivityServers, getWeakSubjectivityState, WeakSubjectivityServer} from "../weakSubjectivityState";
+import {Checkpoint} from "@chainsafe/lodestar-types/phase0";
+import {SLOTS_PER_EPOCH} from "@chainsafe/lodestar-params";
+
+async function initAndVerifyWeakSujectivityState(
+  config: IBeaconConfig,
+  db: IBeaconDb,
+  wsState: TreeBacked<allForks.BeaconState>,
+  logger: ILogger,
+  wsCheckpoint?: Checkpoint
+): Promise<TreeBacked<allForks.BeaconState>> {
+  try {
```

Comment:
I think I messed up here, the error was already being "handled" by just being thrown. My bad :sweat_smile: